.github/workflows/update-showdown-data.yml

```diff
@@ -0,0 +1,40 @@
+name: update-showdown-data
+
+on:
+  schedule:
+  - cron: '0 0 * * *'
```

Comment:
Maybe do it weekly?
As of now these 2 files are updated quite rarely, but in general I'd avoid merging a lot of commits if there's more activity. When new data is released I'd wait a few days anyway since they'll obviously push commits to correct small mistakes.

Reply:
That's a valid point. If you have no further objections I'll probably merge this later today